index.html

```diff
@@ -9,7 +9,81 @@
   </head>
   <body>
     <header>
-      <h1>Quizzy</h1>
+      <img
+        src="/pictures/quiz-comic-pop-art-style_175838-505.jpg"
+        class="logo"
+      />
     </header>
+    <section class="card" class="cardclosed">
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum,
+        cupiditate. Cum officiis consectetur amet ab libero error necessitatibus
+        ea facilis maiores excepturi quidem in, obcaecati, culpa asperiores.
+        Voluptatum, rem dolore?
+      </p>
+      <button type="button" class="buttonshow">Show Answer</button>
+      <ul class="liste">
+        <li><button type="button">#html</button></li>
+        <li><button type="button">#flexbox</button></li>
+        <li><button type="button">#css</button></li>
+      </ul>
+      <img
+        src="/pictures/kisspng-bookmark-computer-icons-like-button-instagram-face-5ada502def3d14.3322865215242568139799.png"
+        class="bookmarkclosed"
+      />
+    </section>
+    <section class="card" class="cardopen">
```

Comment:
die classes kannst du auch in eins schreiben: class="card cardopen" und er erkennt das als zwei verschiedene classes. Spart dir einmal das class tippen ;)